torch/_ops.py

```diff
@@ -845,8 +868,18 @@ def __getattr__(self, key):
             op_, op_dk_, tags = torch._C._get_operation_overload(
                 self._qualified_op_name, use_key
             )
-            schema = torch._C._get_schema(self._qualified_op_name, use_key)
-            overload = OpOverload(self, op_, op_dk_, schema, tags)
+
+            if use_key not in self._schemas:
+                # Raise Runtime error is to be consistant with the error raised by torch._C._get_schema
+                raise RuntimeError(f"Found no matching schema for {use_key}")
+
+            schema = self._schemas[use_key]
+
+            overload = (
+                OpOverload(self, op_, op_dk_, schema, tags)
+                if not self._overload_has_script_obj_arg[use_key]
+                else TorchBindOpOverload(self, op_, op_dk_, schema, tags)
+            )
```

Comment:
We don't need `self._schemas` and `self._overload_has_script_obj_arg`, right? In general we want to avoid the redundancy. If the motivation behind this is caching the attributes for performance: we don't need to do this: `__getattr__` only gets called if the OpOverload can't be found

Reply:
So the logic here should just be:
```py
schema = torch._C._get_schema(self._qualified_op_name, use_key)
            overload = OpOverload(self, op_, op_dk_, schema, 
if has_script_object(schema):
    return TorchBindOpOverload(...)
else:
    return OpOverload
```